ui/components/ui/logo/logo-onekey.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+type LogoOneKeyType = {
+  width: string;
+  color: string;
+  className: string;
+  ariaLabel: string;
+};
+
+const LogoOnekey = ({
+  width = '100%',
+  color = 'var(--color-text-default)',
+  className,
+  ariaLabel,
+}: LogoOneKeyType) => {
```

Comment:
I believe we use `Props` for this, also re-using docs from `logo-ledger.tsx` here:

```suggestion
type LogoOneKeyProps = {
  /**
   * The width of the logo. Defaults to 100%
   */
  width: string;
  /**
   * The color of the logo defaults to var(--color-text-default)
   */
  color: string;
  /**
   * Additional className to add to the root svg
   */
  className: string;
  /**
   * Aria label to add to the logo component
   */
  ariaLabel: string;
};

const LogoOnekey = ({
  width = '100%',
  color = 'var(--color-text-default)',
  className,
  ariaLabel,
}: LogoOneKeyProps) => {
```

Reply:
Why should we use `Props` definitions as we're now using a type instead? Other logo files are still using javascript, not typescript, hence the need of props. BTW, I had to convert to `.tsx` to pass `eslint` job

Reply:
Ah no, sorry I was not clear enough, but for other `.tsx` components, we usually use a `Props` suffix (which is a `type`, I'm not referring to the "JS-way" of declaring those props).

Like see here:
- https://github.com/MetaMask/metamask-extension/blob/b57ed09a7437399cf49402bca3e13981b76315bf/ui/pages/notifications/notifications-list.tsx#L20-L26
- https://github.com/MetaMask/metamask-extension/blob/b57ed09a7437399cf49402bca3e13981b76315bf/ui/pages/notifications/notifications-list.tsx#L107

Reply:
My bad, i read too fast your comment! Indeed, this change makes sense 👍